catalyst/dl/callbacks/metrics/cmc.py

```diff
@@ -197,6 +197,16 @@ def on_loader_start(self, runner: "IRunner"):
 
     def on_loader_end(self, runner: "IRunner"):
         """On loader end action"""
+        assert self._gallery_idx == self._gallery_size, (
+            self._gallery_idx,
+            self._gallery_size,
+        )
+
+        assert self._query_idx == self._query_size, (
```

Comment:
I think, we also need to write some assert message here

Reply:
done